forkable/forkdb.go

```diff
@@ -102,10 +100,8 @@ func (f *ForkDB) TrySetLIB(headRef bstream.BlockRef, previousRefID string, libNu
 //Set a new lib without cleaning up blocks older then new lib (NO MOVE)
 func (f *ForkDB) SetLIB(headRef bstream.BlockRef, previousRefID string, libNum uint64) {
 	if headRef.Num() == bstream.GetProtocolFirstStreamableBlock {
-		f.libRef = bstream.NewBlockRef(previousRefID, bstream.GetProtocolGenesisBlock)
-		libNum = bstream.GetProtocolGenesisBlock
-
-		f.logger.Debug("candidate LIB received is first streamable block of chain, assuming it's the new LIB", zap.Stringer("lib", f.libRef))
```

Comment:
I feel we should keep the log line here, it's an edge case and it make sense to have some report about it.